downloader.py

```diff
@@ -10,6 +10,24 @@
 __log__ = logging.getLogger(__name__)
 
 
+def download_media(client, msg, target_id):
+    media = msg.media
+    os.makedirs('usermedia', exist_ok=True)
+    file_name_prefix = '%d-%d-' % (target_id, msg.id)
+    if isinstance(media, tl.MessageMediaDocument) and not hasattr(media.document, 'stickerset'):
+        file_name = file_name_prefix + next(a for a in media.document.attributes if isinstance(a, tl.DocumentAttributeFilename)).file_name
```

Comment:
That's a rather long line. I'd prefer:
```python
        file_name = file_name_prefix + next(
            a for a in media.document.attributes
            if isinstance(a, tl.DocumentAttributeFilename)
        ).file_name
```

Reply:
By the way, [this is how Telethon does it](https://github.com/LonamiWebs/Telethon/blob/b6053a65ad7fd3f17d3eab835d8e40d85f7d634b/telethon/telegram_client.py#L1549-L1607).

Reply:
Are you happy with my file naming convention though?